fleetspeak/src/inttesting/frr/proto/fleetspeak_frr/frr.proto

```diff
@@ -92,4 +105,14 @@ service Master {
   // by the FS server.
   rpc RecordFileResponse(FileResponseInfo) returns (fleetspeak.EmptyMessage) {
   }
+
+  // CompletedRequests returns a list of requests made to a client which have been
+  // completed.
+  rpc CompletedRequests(ClientID) returns (CompletedRequestsIds) {
```

Comment:
Naming: CompletedRequests(CompletedRequestsRequest) returns (CompletedRequestsResponse)

This naming makes it easier to extend the RPC in the future without touching every single use site, for example if we want to get completed requests based on other criteria than the client_id.